packages/react-components/src/hooks/useAlarms/useAlarms.ts

```diff
@@ -87,6 +88,7 @@ function useAlarms<T>(options?: UseAlarmsOptions<T>): (T | AlarmData)[] {
     iotSiteWiseClient,
     alarmDataList: typePropertyAlarmData,
     alarmPropertyFieldName: 'source',
+    refreshRate: settings?.refreshRate,
```

Comment:
We purposefully didn't set the refresh rate for fetching the alarm type and source because we only need to do that once. We should be setting the refreshRate here to `Infinity` for that reason.

Reply:
oh, then that was a different type of bug then! because it was defaulting to 5s!

Reply:
Changed this to set refreshRate explicitly to Infinity in `useLatestAlarmPropertyValues`